wheelsproxy/client.py

```diff
@@ -12,7 +13,10 @@
 
 from execnet.gateway_base import Unserializer
 
-from .utils import retry_call
+from .utils import retry_call, exponential_backoff
+
+
+MAX_CACHE_BUSTING_RETRIES = 3
```

Comment:
move to settings and try to get the value from the envrionment. we might have to tune this value? how fast is pypi?